src/shogun2-core/src/main/java/de/terrestris/shogun2/model/layer/source/WmtsLayerDataSource.java

```diff
@@ -45,6 +213,14 @@ public int hashCode() {
         // two randomly chosen prime numbers
         return new HashCodeBuilder(11, 19).
             appendSuper(super.hashCode()).
+            append(getFormat()).
+            append(getRequestEncoding()).
+            append(getWmtsLayer()).
+            append(getMatrixSet()).
+            append(getProjection()).
+            append(getWmtsStyle()).
+            append(getTileGrid()).
+            append(getUrls()).
```

Comment:
Maybe we should not use getTileGrid and getUrls here as this may lead to "expensive" DB queries?